Source/Classes/PINRemoteImageDownloadQueue.h

```diff
@@ -0,0 +1,37 @@
+//
+//  PINRemoteImageDownloadQueue.h
+//  PINRemoteImage
+//
+//  Created by Garrett Moon on 3/1/17.
+//  Copyright © 2017 Pinterest. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "PINRemoteImageManager.h"
+
+@class PINURLSessionManager;
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^PINRemoteImageDownloadCompletion)(NSURLResponse *response, NSError *error);
+
+@interface PINRemoteImageDownloadQueue : NSObject
+
+@property (nonatomic, assign) NSUInteger maximumNumberOfOperations;
```

Comment:
does this refer to maxConcurrentOperations?  Maybe better as maxConcurrentDownloads, or connectionSlots, or something else less generic than "operations".

max number feels a bit like things will be dropped from the queue after hitting that number.